app/assets/javascripts/protocols/steps.js.erb

```diff
@@ -673,4 +673,5 @@
     });
   })
 
+  window.initHandsOnTable = initHandsOnTable;
```

Comment:
You can also use global to bind the function to the window object 😄  the window is passed as a param in the IIFE.